homeassistant/components/traccar/device_tracker.py

```diff
@@ -172,11 +159,24 @@ def __init__(self, api, hass, async_see, scan_interval,
             for custom_attr in self._custom_attributes:
                 if device_info.get(custom_attr) is not None:
                     attr[custom_attr] = device_info[custom_attr]
+                    if custom_attr in self._skip_accuracy_on:
+                        skip_accuracy_filter = True
+
+            accuracy = float(0)
+            if device_info.get('accuracy') is not None:
+                accuracy = device_info['accuracy']
+            if (not skip_accuracy_filter and self._max_accuracy > 0 and
```

Comment:
Since `skip_accuracy_filter`  are set inside a loop, and this is used outside.
This type of filtering would not give the expected result (if you have multiple custom_attrs, for 1 it will not matter).

Reply:
Related to previous lines comment, ```skip_accuracy_filter``` var is initialited to ```False``` on line [138](https://github.com/home-assistant/home-assistant/pull/23292/files#diff-5192f6e34a898c79e0b402348c7a7a65R138) outside the loop. It only shold be true when traccar message contains an attribute listed in configuration under ```skip_accuracy_filter_on```.
ie. This configuration should allow to have ```alarm``` and ```last_update``` under ```monitored_conditions``` (custom_attrs) but only ```alarm``` under ```skip_accuracy_filter_on```, so all traccar api reponses retrive ```last_update``` and will be filtered if accuracy is higher than limit, but it will not be filtered if contains an ```alarm```.

Reply:
I see, so the design is that if _any_ attribute is in `_skip_accuracy_on` it should enable `skip_accuracy_filter` for _all_ attributes.

Reply:
Yes!